src/components/card-container/card-container.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class CardContainer extends React.PureComponent {
```

Comment:
Are cards *with* images ever going to coexist with cards *without* images? Would that work, or would it break the layout?